src/components/InfoPanel/infoPanel.module.scss

```diff
@@ -3,35 +3,43 @@
   font-weight: bold;
   background-color: white;
   display: flex;
+  flex-direction: row;
   justify-content: center;
   align-items: center;
   position: absolute;
   bottom: calc(-50% + 75px);
   right: calc(50% - 400px);
   border: 1px solid black;
   border-radius: 10px;
+  z-index: 3000;
 }
 .IPAddress {
+  text-align: center;
   padding: 20px;
   width: 200px;
   height: 150px;
 }
 hr {
-  rotate: 90deg;
-  width: 50%;
+  height: 100px;
 }
 .location {
+  text-align: center;
   padding: 20px;
   width: 200px;
   height: 150px;
 }
 .timezone {
+  text-align: center;
   padding: 20px;
   width: 200px;
   height: 150px;
 }
 .ISP {
+  text-align: center;
   padding: 20px;
   width: 200px;
   height: 150px;
 }
+.params {
+  text-align: center;
```

Comment:
What if we use text-align only once? For `.infoPanel` 😶‍🌫️

Reply:
done